docs/develop/cmodules.rst

```diff
@@ -53,6 +53,30 @@ A MicroPython user C module is a directory with the following files:
   for header files), these should be added to ``CFLAGS_USERMOD`` for C code
   and to ``CXXFLAGS_USERMOD`` for C++ code.
 
+* ``usermod.cmake`` contains the CMake configuration for this module.
+
+  In ``usermod.cmake``, you may use ``${CMAKE_CURRENT_LIST_DIR}`` as the path to
+  the current module.
+
+  Your ``usermod.cmake`` should define an ``INTERFACE`` style library and
```

Comment:
What is an 'interface style library' ? (web search returns no sensible results)

Reply:
https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#interface-libraries

Reply:
> What is an 'interface style library' ? (web search returns no sensible results)

https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#interface-libraries

Reply:
Ok, perhaps just calling it "interface library" would be clearer then.

Reply:
I've made this change, in addition to retrospectively adding some comments to the .cmake files that - hopefully - clarify this. (And since it's mentioned the files include comments... they probably should include comments)

Reply:
> they probably should include comments

Hehe fair indeed.